getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -9,24 +9,33 @@ The following steps detail what is needed to setup the SDK and the engine.
 Download the Android SDK
 ------------------------
 
-Download and install the Android SDK from
-`developer.android.com <https://developer.android.com/studio/>`__.
+Download and install the Android SDK. 
 
-If you install Android Studio, you need to run it once to complete the SDK setup.
+- You can install it using `Android Studio <https://developer.android.com/studio/>`__.
+  
+  - Run it once to complete the SDK setup using these `instructions <https://developer.android.com/studio/intro/update#sdk-manager>`__.
+  - Ensure that the `required packages <https://developer.android.com/studio/intro/update#recommended>`__ are installed as well.
 
-Install OpenJDK 8
+- You can install it using the `command line tools <https://developer.android.com/studio/#command-tools>`__.
+  
+  - Once the command line tools are installed, run the `sdkmanager <https://developer.android.com/studio/command-line/sdkmanager>`__ command to complete the setup process:
+
+::
+
+    sdkmanager --sdk_root=<android_sdk_path> "platform-tools" "build-tools;30.0.1" "platforms;android-29" "cmdline-tools;latest"
```

Comment:
The `platform-tools` seems to already be implicit (from another package?) and this command generates this warning:
```
Warning: Package "com.android.repository.impl.generated.v1.RemotePackage@5f48a0aa" (platform-tools) should be installed in 
"/home/akien/Projects/godot/android/sdk2/platform-tools" but 
it already exists.
Installing in "/home/akien/Projects/godot/android/sdk2/platform-tools-2" instead.
```
$ ls platform-tools* -l
platform-tools:
total 15552
-rwxr-xr-x 1 akien akien 7262264 Jan 12 13:11 adb*
drwxr-xr-x 2 akien akien    4096 Jan 12 13:11 api/
-rwxr-xr-x 1 akien akien   59512 Jan 12 13:11 dmtracedump*
-rwxr-xr-x 1 akien akien 1585992 Jan 12 13:11 e2fsdroid*
-rwxr-xr-x 1 akien akien  309080 Jan 12 13:11 etc1tool*
-rwxr-xr-x 1 akien akien 1942192 Jan 12 13:11 fastboot*
-rwxr-xr-x 1 akien akien   13696 Jan 12 13:11 hprof-conv*
drwxr-xr-x 2 akien akien    4096 Jan 12 13:11 lib64/
-rwxr-xr-x 1 akien akien  258032 Jan 12 13:11 make_f2fs*
-rwxr-xr-x 1 akien akien  257488 Jan 12 13:11 make_f2fs_casefold*
-rwxr-xr-x 1 akien akien  873440 Jan 12 13:11 mke2fs*
-rwxr-xr-x 1 akien akien    1157 Jan 12 13:11 mke2fs.conf*
-rw-r--r-- 1 akien akien  388260 Jan 12 13:11 NOTICE.txt
-rw-r--r-- 1 akien akien   17783 Jan 12 13:11 package.xml
-rwxr-xr-x 1 akien akien 1539624 Jan 12 13:11 sload_f2fs*
-rw-r--r-- 1 akien akien      38 Jan 12 13:11 source.properties
-rwxr-xr-x 1 akien akien 1372144 Jan 12 13:11 sqlite3*
drwxr-xr-x 3 akien akien    4096 Jan 12 13:11 systrace/

platform-tools-2:
total 15552
-rwxr-xr-x 1 akien akien 7262264 Jan 12 13:13 adb*
drwxr-xr-x 2 akien akien    4096 Jan 12 13:13 api/
-rwxr-xr-x 1 akien akien   59512 Jan 12 13:13 dmtracedump*
-rwxr-xr-x 1 akien akien 1585992 Jan 12 13:13 e2fsdroid*
-rwxr-xr-x 1 akien akien  309080 Jan 12 13:13 etc1tool*
-rwxr-xr-x 1 akien akien 1942192 Jan 12 13:13 fastboot*
-rwxr-xr-x 1 akien akien   13696 Jan 12 13:13 hprof-conv*
drwxr-xr-x 2 akien akien    4096 Jan 12 13:13 lib64/
-rwxr-xr-x 1 akien akien  258032 Jan 12 13:13 make_f2fs*
-rwxr-xr-x 1 akien akien  257488 Jan 12 13:13 make_f2fs_casefold*
-rwxr-xr-x 1 akien akien  873440 Jan 12 13:13 mke2fs*
-rwxr-xr-x 1 akien akien    1157 Jan 12 13:13 mke2fs.conf*
-rw-r--r-- 1 akien akien  388260 Jan 12 13:13 NOTICE.txt
-rw-r--r-- 1 akien akien   17783 Jan 12 13:13 package.xml
-rwxr-xr-x 1 akien akien 1539624 Jan 12 13:13 sload_f2fs*
-rw-r--r-- 1 akien akien      38 Jan 12 13:13 source.properties
-rwxr-xr-x 1 akien akien 1372144 Jan 12 13:13 sqlite3*
drwxr-xr-x 3 akien akien    4096 Jan 12 13:13 systrace/
```